mne/io/brainvision/tests/test_brainvision.py

```diff
@@ -452,4 +452,14 @@ def test_events():
     assert_equal(raw.info['chs'][-1]['ch_name'], 'STI 014')
 
 
+def test_brainvision_with_montage():
+    """Test reading embedded montage information"""
+    raw = read_raw_brainvision(vhdr_v2_path, eog=eog, misc=['ReRef'])
+    for i, d in enumerate(raw.info['dig'], 1):
+        assert_equal(d['coord_frame'], FIFF.FIFFV_COORD_HEAD)
+        assert_equal(d['ident'], i)
+        assert_equal(d['kind'], FIFF.FIFFV_POINT_EEG)
+        assert_equal(len(d['r']), 3)
```

Comment:
It should not affect the location fields no. I could add something like

    raw_none = read_raw_brainvision(vhdr_v2_path) 
    for r,n in zip(raw.info['dig'], raw_none.info['dig']):
        assert_array_equal(r['r'], n['r'])
                
    for r,n in zip(raw.info['chs'], raw_none.info['chs']):
        assert_array_equal(r['loc'], n['loc'])

though to test if positions are the same. I you think so.